.travis.yml

```diff
@@ -1,3 +1,13 @@
-install: gem install kramdown
+language: python
+python:
+  - "2.7"
+sudo: false
+install:
+  - gem install kramdown
 
-script: make
+  - git clone --depth=1 --branch=master https://github.com/tabatkins/bikeshed.git ./bikeshed
+  - pip install pygments
+  - pip install --editable ./bikeshed
+  - bikeshed update
+script:
+  make
```

Comment:
It's supposed to [build the bikeshed version](https://github.com/w3c/webappsec-subresource-integrity/blob/68f19599a2392db2606ff7386502b204a588da1d/Makefile#L1) but as we can see from the [travis output](https://travis-ci.org/w3c/webappsec-subresource-integrity/builds/129305471#L206), it only rebuilds the kramdown version.

I suspect the problem is that we're not deleting `index.bikeshed.html` in the [clean target](https://github.com/w3c/webappsec-subresource-integrity/blob/68f19599a2392db2606ff7386502b204a588da1d/Makefile#L4).